CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing Guidelines  
+  
+This documentation contains a set of guidelines to help you during the contribution process.   
+We are happy to welcome all the contributions from anyone willing to improve/add new scripts to this project. Thank you for helping out and remember, **no contribution is too small.**  
+  
+# Submitting Contributions👩‍💻👨‍💻  
+Below you will find the process and workflow used to review and merge your changes.  
+
+## Step 0 : Find an issue  🔍
+- Take a look at the Existing Issues or create your **own** Issues!  
+- Wait for the Issue to be assigned to you after which you can start working on it.  
+- Note : Every change in this project should/must have an associated issue.   
+![issue](https://github.com/kanishkaa24/Community-Website/blob/main/Untitled%20design%20(6).png)  
+  
+## Step 1 : Fork the Project 🍴
+- Fork this Repository. This will create a Local Copy of this Repository on your Github Profile. Keep a reference to the original project in `upstream` remote.  
+```  
+$ git clone https://github.com/<your-username>/<repo-name>  
+$ cd <repo-name>  
+$ git remote add upstream https://github.com/<upstream-owner>/<repo-name>  
```

Comment:
please change this to `git remote add upstream https://github.com/HITK-TECH-Community/Community-Website`